### ISSUE TITLE
[Bug]: Cannot start nuxt: Cannot find module 

### ISSUE THREAD

**Issue by robmontesinos** [stefanobartoletti/nuxt-social-share#283]
### Please take a moment to familiarize yourself with this project scope and standards

- [X] I have read the [Documentation](https://github.com/stefanobartoletti/nuxt-social-share?tab=readme-ov-file) and I understand how to correctly use this module.
- [X] I have read the [Contributing Guidelines](https://github.com/stefanobartoletti/nuxt-social-share/blob/main/.github/CONTRIBUTING.md).
- [X] I agree to follow this project's [Code of Conduct](https://github.com/stefanobartoletti/nuxt-social-share?tab=coc-ov-file)

### Environment

```shell
- Operating System: `Darwin`
- Node Version:     `v21.5.0`
- Nuxt Version:     `3.13.2`
- CLI Version:      `3.14.0`
- Nitro Version:    `2.9.7`
- Package Manager:  `pnpm@9.12.2`
- Builder:          `-`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`
```

### Minimal reproduction link

No

### Describe the bug/issue

Upon first relaunch of dev server since last nuxt-social-share update ("@stefanobartoletti/nuxt-social-share": "^1.1.4",), we got the "Cannot start nuxt" error. We have to remove this module to continue developing but hope that you can fix the issue. We tried removing and reinstalling module, deleting node_modules and pnpm-lock.yaml to no avail.

### Logs

```shell
Error while requiring module @stefanobartoletti/nuxt-social-share: Error: Cannot find module '/home/stefano/Sviluppo/Personali/nuxt-social-share/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/lib/index.js'
Require stack:
- /Users/xxxx/xxxxxx/xxxxxxx/node_modules/.pnpm/@stefanobartoletti+nuxt-social-share@1.1.4_magicast@0.3.5_rollup@4.24.0_webpack-sources@3.2.3/node_modules/@stefanobartoletti/nuxt-social-share/dist/module.mjs


[6:42:34 PM]  ERROR  Cannot start nuxt:  Cannot find module '/home/stefano/Sviluppo/Personali/nuxt-social-share/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/lib/index.js'
Require stack:
- /Users/xxxx/xxxx/xxxxx/node_modules/.pnpm/@stefanobartoletti+nuxt-social-share@1.1.4_magicast@0.3.5_rollup@4.24.0_webpack-sources@3.2.3/node_modules/@stefanobartoletti/nuxt-social-share/dist/module.mjs
```

**Comment by YannMLD** [comment]
I was able to relaunch the app by downgrading to 1.1.2

**Comment by stefanobartoletti** [comment]
Thanks for reporting this, but I'm unable to reproduce the issue. Testing it on multiple sites of mine, everything is working fine.

Without a minimal reproducible environment, unfortunately I am unable to help you.

**Comment by stefanobartoletti** [comment]
Ok, on a closer inspection, apparently the package was incorrectly built in `v1.1.4`, but everything should be solved now on `v1.1.5`

@robmontesinos @YannMLD can you please test and confirm that the latest version is working correctly again?

**Comment by robmontesinos** [comment]
@stefanobartoletti Thank you so much Stefano. My first step was 'pnpm add @stefanobartoletti/nuxt-social-share' and version 1.1.4 was installed according to package.json and the same errors were generated.

I then entered 'pnpm add @stefanobartoletti/nuxt-social-share@latest' which installed version 1.1.5 and Bang Zoom!! It worked perfectly like before. Grazie Stefano - great work and so fast.
